schemas/rest-api/search-results.json

```diff
@@ -0,0 +1,19 @@
+{
+	"$schema": "http://json-schema.org/draft-07/hyper-schema#",
+	"$id": "https://raw.githubusercontent.com/johnbillion/wp-json-schemas/trunk/schemas/rest-api/search-results.json",
+	"title": "WP_REST_API_Search_Results",
+	"description": "A collection of search result objects in a REST API context.",
+	"type": "array",
+	"items": {
+		"$ref": "search-result.json"
+	},
+	"links": [
```

Comment:
@johnbillion I wasn't quite sure how to handle the `links` portion of the schema, so please review and perhaps clarify how links should be handled in contributing guidelines. I did compare other schemas `links` to the items found in the `_links` portion of their corresponding `OPTIONS` request, however I often found that the `OPTIONS` request included more `_links` items than the schema did.

Reply:
The `_links` property is something I haven't gotten around to adding full schema information for. The existing `object-links.json` schema describes the shape of any `_links` property, but not the actual link elements for any endpoints.